### DISCUSSION TITLE
GnssRinexNavigation2OrbitClock for GLONASS and SBAS

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#144] [Q&A] [answered]
Dear Torsten, 

In converting the RINEX navigation files to orbits and clocks, I have noticed that there is an issue to converting the GLONASS and SBAS satellite data. By reading the BRDM files, I manged to get the orbits and clocks for GPS and Galileo, but due to the following error message, I could not succeed to get the GLONASS and SBAS orbits and clocks. This the error message:

**(process  4) conversion not implemented yet for these satellites: R01 R02 R03 R04 R05 R06 R07 R08 R09 R10 R11 R12 R13 R14 R15 R16 R17 R18 R19 R20 R21 R22 R23 R24 R26 S22 S23 S25 S26 S27 S28 S29 S30 S31 S32 S33 S36 S37 S38**

I think there is a bug in this program which inhibits to convert the GLONASS and SBAS orbits and clock.

Thank you for your time and support. 

With kind regards, 
Parisa

**Comment by akvas** [comment, marked as answer]
See issue #145.
